Exercise_B/index.html

```diff
@@ -15,8 +15,11 @@
     />
   </head>
   <body>
-    <!-- CODE HERE -->
-
+    <Div>
+      <Div class="Blue"></Div>
+      <Div class="Green"></Div>
```

Comment:
Do not make HTML elements start with an uppercase letter
Classes should also not start with uppercase letters